aips/aip-native-utilities.md

```diff
@@ -0,0 +1,179 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Value manipulation move stdlib native utilities
+author: igor-aptos
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: <Standard (Core, Networking, Interface, Application, Framework) | Informational | Process>
+created: <mm/dd/yyyy>
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-X - Value manipulation move stdlib native utilities
```

Comment:
not sure if it's just me, I don't feel "manipulation" is a good word to use, maybe "operation"?

Reply:
I don't know - seems like a common term - searching on google I get:

"Programming language value manipulation" refers to the process of changing or altering the data stored in variables within a programming language, using operators and functions